feat: Add removal release note for repository integrations CY-7215 (#1944)
codacy/docs: docs/release-notes/cloud/cloud-2023-11-13-jira-slack-webhooks-repo-integrations-removal.md

---
rss_title: Codacy release notes RSS feed
rss_href: /feed_rss_created.xml
---

# Removal of Jira, Slack, and Webhooks repository integrations November 13, 2023

On the week of November 13th 2023 we removed the option to create, configure, and delete **Jira**, **Slack**, and **Webhooks integrations for your repositories**. These integrations were available on the repository **Settings**, tab **Integration**.

> Existing integrations will keep working for now, we'll remove them on a second phase. However, you're no longer able to configure or delete them on the Codacy app.

Reviewer: Small edit to remove the comma splice and the use of we:

```suggestion
Existing integrations will keep working for now, and they will be deactivated at a later date. However, you're no longer able to configure or delete them on the Codacy app.
```

> Existing integrations will keep working for now, we'll remove them on a second phase. However, you're no longer able to configure or delete them on the Codacy app.

Reviewer: We could also either:

1) Link the "deactivated at a later date" to the [Removal](https://github.com/codacy/docs/pull/1944/files#diff-f96c2c2dc04b493018f9ab789dd5a9c772179eb4fb614a262eeeb8d4045e98b9R14) section
2) Specify the deactivation date here as well when we know it.

> Existing integrations will keep working for now, we'll remove them on a second phase. However, you're no longer able to configure or delete them on the Codacy app.

Author: I applied both options (link and specify date). I believe they work well together, as we have more details in that section.

Also, I'm keeping "remove" instead of "deactivate" for consistency.

```suggestion
Your existing integrations will keep working for now and [will be removed on the week of MM DD YYYY](#removal-of-existing-integrations). However, you can no longer configure or delete them on the Codacy app.<!--TODO CY-7215 Update date-->
```

> Due to <span class="skip-vale">very</span> low usage of these integrations, their maintenance had been neglected for years. Thus, they were starting to negatively impact our speed to deliver new features. Although there's no timeline in place to make these integrations available again, we're open to looking into them upon feedback.

Author: Let's keep this section to explain the rationale and the intent to reevaluate, as keep using the "If you are using..." section at the bottom. This ensures consistency across these deprecation/removal notes (following a template).
   
```suggestion
Due to their limited usage, these integrations haven't been maintained for some time and have started impacting the delivery of new features. Although there's no plan to make these integrations available again, we're open to looking into them upon feedback.
```

---
rss_title: Codacy release notes RSS feed
rss_href: /feed_rss_created.xml
---

# Removal of Jira, Slack, and Webhooks repository integrations November 13, 2023

On the week of November 13th 2023 we removed the option to create, configure, and delete Jira, Slack, and Webhooks integrations for your repositories. These integrations were available on the repository **Settings**, tab **Integration**.

> <!--TODO CY-7215 Add rationale for this removal-->

Author: @RaquelHipolito @mrfyda I need your help here... How can we explain to our customers the rationale for removing these integrations?

Reviewer 2: be honest 😁 

these integrations had low adoption, as a consequence their maintenance had been neglected for years, and ultimately they we making shipping new features slower.
that being said, it's not that we don't believe in the problems they were solving, we just weren't happy with the current implementations. while there is no timeline in place to make them available again, we're open to looking into them again

## Removal of existing integrations

> On the week of MM DD YYYY we'll be removing your existing integrations.<!--TODO CY-7215 Update date to remove existing integrations-->

Reviewer 2: maybe @inesgomas has better visibility

we don't need an accurate date, just a ballpark that we can adjust to when the migration is code-ready

however, the expected date is a nice-to[have in this case: there are no actions to be taken, or alternatives to implement.

> On the week of MM DD YYYY we'll be removing your existing integrations.<!--TODO CY-7215 Update date to remove existing integrations-->

Reviewer 3: I proposed to the team that tomorrow we refine those cleanups

> On the week of MM DD YYYY we'll be removing your existing integrations.<!--TODO CY-7215 Update date to remove existing integrations-->

Reviewer 4: As @mrfyda mentioned perhaps we can release this with a timeframe of end of the year timeline and add the detail of "further comms on this will be sent via email, make sure you have them enable (with link to the email permitions page.)"

> On the week of MM DD YYYY we'll be removing your existing integrations.<!--TODO CY-7215 Update date to remove existing integrations-->

Author: [As agreed](https://github.com/codacy/docs/pull/1944#pullrequestreview-1754830151), we'll later add the action to re-subscribe to marketing emails.

> <!--TODO CY-7215 Impacts?-->

Author: @RaquelHipolito @mrfyda Any impacts to mention, besides that the integrations will stop working?

Reviewer 2: that's it. the integrations and the features they empower

## If you are using Jira, Slack, or Webhooks repository integration

Your existing integrations will keep working until MM DD YYYY, but you are no longer able to configure, or delete them on the Codacy app.

> <!--TODO CY-7215 Instructions for customers?-->

Author: @RaquelHipolito @mrfyda For customers using these integrations, are there any particular steps they can or should do?

Reviewer 2: no actions required

if you want to add a note, we'll also delete any authentication data that is left from the integrations once they are removed

if they have a critical workflow relying on these integrations, please let us know what so that we can advise on the best alternative
